app/helpers/sessions_helper.rb

```diff
@@ -15,6 +18,11 @@ def remember(user)
     user.remember
     cookies.permanent.encrypted[:user_id] = user.id
     cookies.permanent[:remember_token] = user.remember_token
+    cookies.permanent[:access_token] = Api::AccessToken.create(email: EMAIL, password: PASSWORD).value if user.admin?
+  end
+
+  def verify_access_token?
+    !Api::AccessToken.new(value: cookies[:access_token]).expired?
```

Comment:
`!expired?` という表現は読みづらいですね。
行の先頭と末尾のふたつを組み合わせて条件を理解する必要があって分かりづらいです。
(そもそも否定文があることで読みづらさが増える気がしますね)

ですから、expired? の逆の判定を用意できるといいですよね
```suggestion
    Api::AccessToken.new(value: cookies[:access_token]).valid?
```